### PR TITLE
Refs [TASKS][25828] Implement comments for campaign

### PR DESCRIPTION
**Pull request by hoatb-1022**
https://edu-redmine.sun-asterisk.vn/issues/25828

Screen Shot 2020-07-21 at 08 04 53

### CODE REVIEW

**Comment by hoatb-1022** [comment]
ready

**Comment by hoatb-1022** [comment]
ready

File: app/javascript/packs/comments.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+    $(`#comment_${commentId} .submit-comment .btn-cancel-edit`).click(function () {
+      editCommentFormEl.addClass('d-none')
+      commentContentEl.removeClass('d-none')
+
+      let commentTextAreaEl = $(`#comment_${commentId} .edit_comment .form-comment textarea`)
+      commentTextAreaEl.val(commentTextAreaEl.attr('origin-val'))
+    })
+  })
+}
```
Comment (manhnt-0960) [inline app/javascript/packs/comments.js:20]: endline

File: app/views/campaigns/show.html.erb [hunk 2]
```diff
@@ -148,8 +148,11 @@
           </div>
         </div>
         <div class="my-4">
-          <%= link_to t(".donate_now"), donate_path(@campaign), class: "btn btn-block btn-primary" %>
-          <%= link_to t(".share_now"), "#", class: "btn btn-block btn-danger" %>
+          <%= link_to t(".donate_now"), new_campaign_donation_path(@campaign), class: "btn btn-block btn-primary" %>
+          <%= link_to t(".share_now"), "https://www.facebook.com/sharer/sharer.php?u=#{request.url}",
```
Comment (manhnt-0960) [inline app/views/campaigns/show.html.erb:152]: https://www.facebook.com/sharer/sharer.php?u= đưa vào settinsg

**Comment by manhnt-0960** [comment]
fix vào pull sau nhé

**Pull request merged**